pg_search/tests/replication.rs

```diff
@@ -167,8 +167,7 @@ impl EphemeralPostgres {
 }
 
 // Test function to test the ephemeral PostgreSQL setup
-#[rstest]
-#[ignore = "@@@"] // Segfaults on PG13.
+#[rstest] // Segfaults on PG13.
```

Comment:
oh, I don't have the ability to check this segfaulting on pg13 but I bet it's just related to the segfault fixed in this PR.

Reply:
Seems to work in CI, we should remove the comment